### ISSUE TITLE
Feature Request - Add proxies for email verification

### ISSUE THREAD

**Issue by ashleykleynhans** [drallieiv/KinanCity#53]
Now that email verification is rate limited, it would be great if you could add a feature where the mail server can read an optional config of proxies to get around the rate limit issue.

**Comment by drallieiv** [comment]
Until mail and core are united in a single service, implemeting proxies on both modules will still cause issues if the 2 modules share the same IPs. The 503 errors thrown by PTC fronts are shared for both creation and activation.

Currently, the creation with core has a server side limit of 5 accounts per 16 minutes, which makes around 20/h. 
While the validation server has a limit of 6 per minute, which is around 360/hour

So unless you are using more then 18 IPs, the mail server is faster then the account are produced.

**Comment by ashleykleynhans** [comment]
Yeah, I'm using lots of IPs which is why I encountered this issue.
